gnoi/reset/client_test.go

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package reset
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/google/gnxi/gnoi/reset/pb"
+)
+
+func TestCheckResponse(t *testing.T) {
+	tests := []struct {
+		response *pb.StartResponse
+		want     *ResetError
+	}{
+		{
+			response: initializeResponse(true, false, false, ""),
```

Comment:
just add directly here the following, it is easier to read:

```
response: &pb.StartResponse{&pb.StartResponse_ResetError{&pb.ResetError{
			FactoryOsUnsupported: true,
			ZeroFillUnsupported:  false,
			Other:                false,
			Detail:               "",
		},},},
```

Reply:
Sure, no problem, will change